packages/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/errors/DevLauncherUncaughtExceptionHandler.kt

```diff
@@ -84,4 +92,32 @@ class DevLauncherUncaughtExceptionHandler(
       }
     }
   }
+
+  private fun tryToSendExceptionToBundler(exception: Throwable) {
+    if (
+      controller.mode != DevLauncherController.Mode.APP ||
+      !controller.appHost.hasInstance() ||
+      controller.appHost.reactInstanceManager.currentReactContext === null
+    ) {
+      return
+    }
+
+    try {
+      val url = Uri
+        .parse(controller.appHost.reactInstanceManager.devSupportManager.sourceUrl)
+        .buildUpon()
+        .path("logs")
+        .clearQuery()
+        .build()
```

Comment:
it would be better to use the `logUrl` property in the manifest rather than hardcoding this construction. (and if the manifest has no `logUrl`, or we have no manifest, then we shouldn't try to send this.)